utils/docker-compose/node-sdk/invoke.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict';
+
+const { FileSystemWallet, Gateway } = require('fabric-network');
+const fs = require('fs');
+const path = require('path');
+
+const ccpPath = path.resolve(__dirname, 'connection.json');
+const ccpJSON = fs.readFileSync(ccpPath, 'utf8');
+const ccp = JSON.parse(ccpJSON);
+
+async function main() {
+    try {
+        var user = process.argv[2]
+        var channel = process.argv[3]
+        var chaincode = "ecc_" + process.argv[4]
```

Comment:
As we still have the USE_FPC option, above wouldn't really work if we are not using FPC?

Reply:
Yes, that is correct. For now I have made a note in the README that if users choose to use a non-fpc network most of the tutorial will not work. But I can make changes in a followup commit/pr.